python-sdk/tests_integration/astro_deploy/deploy.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
```

Comment:
Can't we deploy using astro-cli? I think that is safer and we don't need to go into details that much e.g. doing http requests.

Reply:
@pankajastro @rajaths010494, did you consider the possibility of using the Astro CLI? I'm curious if this approach was considered or ruled out for any reason.

Since this PR has been long-standing, could we also have this as a separate ticket (to change the deployment to use Astro CLI) and try to close this PR ASAP - WDYT?

Reply:
We had used the same approach in providers repo so we used the same here. If already ticket exists for enhancement. If not we can create one

Reply:
This script adds a lot of maintenance for us. Let us please implement the github action or cli, but not a script from scratch. I know this is being used in the providers repo. We should also replace this there.

Reply:
Yeah sure will create a ticket in provider repo as well to replace the script.

Reply:
I tried to use it around 10 months back in the provider repo but at that time we had some roadblocks in making it work with full automation for custom images. But, I think it makes sense to look at docs and retry since lots have changed since then.

Reply:
Created a ticket to use astro-deploy in a separate ticket: https://github.com/astronomer/astro-sdk/issues/1742
cc: @rajaths010494 @phanikumv